tests/Examples/tfhe_rust_hl/cpu/BUILD

```diff
@@ -43,5 +45,6 @@ tfhe_rs_end_to_end_test(
     ],
     heir_translate_flags = ["--emit-tfhe-rust-hl"],
     mlir_src = "hello_world_clean_xsmall.mlir",
+    tags = ["manual"],
```

Comment:
same here to eventually remove manual https://github.com/google/heir/issues/1721